templates/_base-images-ref.html

```diff
@@ -0,0 +1,461 @@
+<script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+
+  (function () {
+    window.jQuery
+      .ajax({
+        url: "https://api.balena-cloud.com/v6/device_type?$select=name,slug&$expand=is_of__cpu_architecture($select=slug)&$filter=is_default_for__application/any(idfa:((idfa/is_host%20eq%20true)%20and%20(idfa/is_archived%20eq%20false))%20and%20not(contains(idfa/app_name,%27esr%27)))&$orderby=name%20asc"
+      })
+      .done(function (deviceTypes) {
+
+        var IGNORE_THESE_DEVICES = ['n310-tx2', 'apalis-imx6', 'colibri-imx6', 'cubox-i', 'edge', 'etcher-pro', 'edison', 'hummingboard2', 'odroid-u3+', 'odroid-ux3', 'nuc', 'parallella-hdmi-resin', 'solidrun-imx6']
```

Comment:
Heads-up that only the following are in the list of returned items: 'n310-tx2', 'etcher-pro'.
Any chance the rest are just aliases? eg `nuc` is an alias and the actual DT slug is `intel-nuc`
Here is the actual slug list:
```
["n310-tx2", "apalis-imx6q", "colibri-imx6dl", "hummingboard", "edge", "etcher-pro", "intel-edison", "hummingboard", "odroid-xu4", "odroid-xu4", "intel-nuc", "parallella", "hummingboard"]
```
which I got by doing
```
await Promise.all(['n310-tx2', 'apalis-imx6', 'colibri-imx6', 'cubox-i', 'edge', 'etcher-pro', 'edison', 'hummingboard2', 'odroid-u3+', 'odroid-ux3', 'nuc', 'parallella-hdmi-resin', 'solidrun-imx6'].map(dt => sdk.models.device.getDeviceSlug(dt)))
```

Now the extra question on top of this is why do we actually need to exclude those?
Probably better to add as a comment so that we don't forget.